### PR TITLE
feat: Add support for api token management

### PR DESCRIPTION
**Pull request by toppercodes**

### CODE REVIEW

File: axiom/provider_test.go [hunk 1]
```diff
@@ -173,19 +191,19 @@ func testAccAxiomDatasetConfig_basic(orgID string) string {
 		name        = "terraform-provider-dataset"
 		description = "A test dataset"
 	}
-
+
```
Comment (Rambatino) [inline axiom/provider_test.go:194]: 👀

**Review by Rambatino — APPROVED** [review]
Tests?

File: axiom/provider_test.go (hunk 1 above)
Comment (toppercodes) [reply axiom/provider_test.go:194]: Tests are there

Comment (toppercodes) [reply axiom/provider_test.go]: I ran terraform fmt on the test terraform

**Pull request merged**